packages/preview/vlna/0.1.0/README.md

```diff
@@ -0,0 +1,10 @@
+# typst-vlna
+
+First version of vlna. Main goal is to create similiar package to luavlna/vlna from LaTeX.
+
+## How to use:
+````typst
+#import "@preview/vlna:0.1.0":*
```

Comment:
I'm not sure this would work: the `show` rule would only apply to the scope in which it is defined, which means the file in which it is defined. What you probably want to do instead is to wrap it in a function that takes a `body` parameter, applies the show rule and returns the body as it was (but with the rule applied).

Reply:
Yes that should work.